chromadb/test/db/test_system.py

```diff
@@ -20,6 +21,23 @@
 from pytest import FixtureRequest
 import uuid
 from chromadb.api.configuration import CollectionConfigurationInternal
+import logging
+import time
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+# Decorator to log the start and end of each test function
+def log_test(func):
```

Comment:
should we make this a more widely accessible util rather than shoving it into this specific file?